docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -1076,7 +1066,7 @@ import jakarta.ws.rs.Path;
 import org.jboss.resteasy.reactive.RestForm;
 import org.jboss.resteasy.reactive.RestPath;
 
-@Path("/project")
+@Path("/project/{projectName}")
 public class ProjectResource {
 
     @PermissionsAllowed("rename-project") <1>
```

Comment:
Line 1122 says `TIP: Permission checks run by default on event loops whenever possible.`, as you are already fixing permission documentation, maybe you could fix that as well, please? I think I forgot to drop `whenever possible` as Clement wanted. Sorry.

Reply:
We can do it later if you don't want.

Reply:
@michalvavrik I've np at all, may be it is worth getting a small doc PR out only for `{projectName}` and this minor update so that we can backport ?

Reply:
these 2 requests come from `@PermissionChecker` so they cannot be backported. I don't mind opening small PR, but it will be tiny (probably this one tip). I'm fine anyway.